assets/robotest/current/status.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: echoserver-status
+  namespace: default
+spec:
+  template:
+    metadata:
+      name: echoserver-status
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: echoserver-status
+          image: quay.io/gravitational/debian-tall:buster
+          command:
+            - sh
+            - -c
+            - wget --quiet --output-document=- echoserver:$ECHOSERVER_SERVICE_PORT
```

Comment:
Is this still going to print an error if the query fails even with the quiet flag set?

Reply:
Good point.  I chose quiet because of `--output-document=-`  Having wget's verbose (default) output interlaced was unsightly.  I changed this to `--no-verbose` in 2a82ca9.